portfolio/src/test/java/com/google/sps/TimeRangeTest.java

```diff
@@ -0,0 +1,227 @@
+package com.google.sps.data;
+
+import java.time.Duration;
+import java.time.Instant;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class TimeRangeTest {
+
+  private static Duration DURATION_30_MINUTES = Duration.ofSeconds(30 * 60);
+
+  private static Instant TIME_0900 = Instant.parse("2020-06-25T09:00:00Z");
+  private static Instant TIME_0930 = TIME_0900.plus(DURATION_30_MINUTES);
+
+  /** Makes sure that the start() method works properly. */
+  @Test
+  public void startMethod() {
```

Comment:
I think it's a bit unusual to include "Method" in the method name. You can rename for example to testStart or test_start. Here and below.

Reply:
Updated a few test function names.